fc/auth/fileauth.py

```diff
@@ -4,7 +4,7 @@
 
 
 class FileAuth(Auth):
-    _file = '~/.fcli'
+    _file = '~/.fcli/.fcli-creds'
```

Comment:
What was the point of moving the file location?  I don't see anything preventing the credentials file still being `~/.fcli`.